### PR TITLE
added asin.md

### LINKED ISSUE #2203: [Term Entry] PHP Math Functions asin()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `asin()` function.

The entry should go in a new file in `docs/content/php/concepts/math-functions/terms/asin/asin.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by ChiragAgg5k**
### Description

Added a new term entry for php about the asin() function in a new file in docs/content/php/concepts/math-functions/terms/asin/asin.md

Closes #2203

### Type of Change

- Adding a new entry

### Checklist

- [✅] All writings are my own.
- [✅] My entry follows the Codecademy Docs style guide.
- [✅] My changes generate no new warnings.
- [✅] I have performed a self-review of my own writing and code.
- [✅] I have checked my entry and corrected any misspellings.
- [✅] I have made corresponding changes to the documentation if needed.
- [✅] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✅] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✅] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--

### CODE REVIEW

File: content/php/concepts/math-functions/terms/asin/asin.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```pseudo
+asin(number)
+```
+
+The `asin()` function takes only one parameter, `number`, a number between -1 and 1 of type `float`. The function returns the arc sine of the `number` in radians as a `float` value.
+
+If the `number` is outside of the range of -1 to 1, the function returns `NAN` (not a number).
+
+
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/asin/asin.md:29]:
```suggestion
```
Remove extra line break.

File: content/php/concepts/math-functions/terms/asin/asin.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## Codebyte Examples
+
+The codebyte example below uses the `asin()` function to return the arc sine of the `2` and `-1`:
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/asin/asin.md:32]:
```suggestion
The codebyte example below uses the `asin()` function to return the arc sine of `2` and `-1`:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ChiragAgg5k I did a first review and left two comments. Otherwise this looks good.

**Comment by ChiragAgg5k** [comment]
Implemented the reviewed changes

**Comment by ChiragAgg5k** [comment]
@SSwiniarski hey... is there any update regarding this pr?

**Comment by yangc95** [comment]
Thanks for your patience @ChiragAgg5k I can conduct the second review.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
🥳 @ChiragAgg5k live link: https://www.codecademy.com/resources/docs/php/math-functions/asin

Screenshot 2023-06-05 at 7 25 21 PM

It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
To add your GitHub username to your Codecademy profile:
1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
githubusername

Make sure to set your visibility to “everyone”:

Screenshot 2023-04-10 at 4 07 30 PM

2.) Then log out and log back into Codecademy using your GitHub login:
githublogin

If the steps above do not work, it may be due to an email or local configuration issue.
In addition, we may reach out in the future if any adjustments to these entries are required.


cc: @SSwiniarski

**Pull request merged**